assignment06.cpp

```diff
@@ -0,0 +1,84 @@
+/*-----------------------------------------------------------------------
+ *              Name: Thanh Phan
+ *              Course: CPSC 121, Tue Lab
+ *              Professor: Ben Blazak
+ * -----------------------------------------------------------------------*/
+/* -----------------------------------------------------------------------
+ * Copyright &copy; 2015 Thanh Phan <tphan2650@gmail.com
+ * Released under the [MIT License] (http://opensource.org/
+ * licenses/MIT)
+ * -----------------------------------------------------------------------*/
+#include <iostream>
+using std::cout;
+using std::endl;
+using std::ostream;
+
+#include <vector>
+using std::vector;
+
+ostream& operator<<(ostream& os, const vector<int> &v) 
+{
+	os << "( ";
+	for (int i = 0; i < v.size(); i++) 
+	{
+		os << v[i] << " ";
+	}
+	os << ")";
+
+	return os;
+}
+
+//Selection Sort
+void print_array(int arr_size, const int array[])
+{	
+	cout << "(";
+
+	for(int i=0;i<arr_size;i++)
+	{
+		cout << array[i];
+	}
+
+	cout << ")";
+}
+
+void selection_sort_in_place(int arr_size, int array[])
+{
+	int i,j,k;
+	int min;
+
+	for(i=0;i<arr_size-1;i++)
+	{
+		k=i; min=array[k];
+		for (j=i+1;j<arr_size;j++)
+			if (array[j]< min)
+			{
+				k=j; min=array[k];
+			}
+			array[k]=array[i]; array[i]=min;
```

Comment:
good practice to have only one statement per line (at least, in cases like this)